api/HTMLElement.json

```diff
@@ -1540,16 +1540,22 @@
             "edge": {
               "version_added": "79"
             },
-            "firefox": {
-              "version_added": "77",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "dom.forms.inputmode",
-                  "value_to_set": "true"
-                }
-              ]
-            },
+            "firefox": [
```

Comment:
Note, I can't find anything about `HTMLElement.inputMode` in the spec - am I missing anything? It "feels" like this perhaps is an old name for the new global attribute, and perhaps has no independent existance?

Reply:
https://html.spec.whatwg.org/#editing-2 has WebIDL defines